### DISCUSSION TITLE
Filtering Annotation Layers

### DISCUSSION THREAD

**Discussion by bluepython508** [apache/superset#18744] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have a chart where I need to show data that can be filtered alongside medians of the unfiltered dataset. However, I also need to hide some or all of these medians on the dashboard.

**Describe the solution you'd like**
Ideally, a way to specifically target annotation layers with dashboard native filters.

**Describe alternatives you've considered**
- Using a mixed time-series chart
  This would not really allow filtering the dataset without also filtering the medians, and would require the medians to be stored in the same dataset.
- A legend, similar to the ECharts legends that allow hiding series, for annotation layers as a whole.

**Additional context**

**Comment by srinify** [comment]
Hey @bluepython508 I'm moving this over to Github discussions because we'd like to keep Issues focused on true bugs / bugs to be validated or fixed! Thank you :)

**Reply by bluepython508** [reply]
So why do you offer a feature request template in the issues section?
